apps/dashboard/app/(app)/logs-v2/components/table/logs-table.tsx

```
@@ -122,80 +101,104 @@ export const LogsTable = ({ onLogSelect, selectedLog }: Props) => {
+                log.response_status >= 400 &&
+                  log.response_status < 500 &&
+                  WARNING_ICON_STYLES.warning,
+                log.response_status >= 500 && WARNING_ICON_STYLES.error
+              )}
+            />
+            <TimestampInfo
+              value={log.time}
+              className={cn(
+                "font-mono group-hover:underline decoration-dotted",
+                selectedLog &&
+                  selectedLog.request_id != log.request_id &&
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)